libbeat/docs/processors-using.asciidoc

```diff
@@ -215,7 +215,6 @@ ifdef::has_decode_csv_fields_processor[]
  * <<decode-csv-fields,`decode_csv_fields`>>
 endif::[]
  * <<decode-json-fields,`decode_json_fields`>>
- * <<decode-base64-field,`decode_base64_field`>>
```

Comment:
Why did you remove this reference?

Reply:
It is duplicated :-) I just removed the second reference.